scripts/Recovery/GenerateMess.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import print_function
```

Comment:
Was this file meant to be added as it looks a bit like some example code or a test?